### PR TITLE
more consisten fonts and less redunancy in CSS

### PR DESCRIPTION
**Pull request by leem42**
making prototypeVue branch up to date with master

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -3,7 +3,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (liujoshua) [inline .editorconfig]: This contradicts js lint and the fact most files are indented by 2. We should either leave this at 2 or switch everything to 4.

**Review by liujoshua — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (leem42) [reply .editorconfig]: Ah, didn't read that correctly when I was merging, I can leave it at two

File: src/components/Eligibility.vue [hunk 2]
```diff
@@ -22,7 +22,7 @@
       <!--Field input two-->
       <div v-if="isUnderage !== null && !isUnderage" class="input-group col-md-4">
         <label class="form-input-label mr-2 inputLabel"> I live in </label>
-        <input v-model="zipCode"  id="placeField" class="form-control inputLabel" type="number" pattern="\d*" placeholder="enter 5-digit zip"></input>
+        <input v-focus="isUnderage" v-model="zipCode"  id="placeField" class="form-control inputLabel" type="number" pattern="\d*" placeholder="enter 5-digit zip"></input>
```
Comment (liujoshua) [inline src/components/Eligibility.vue]: When place gets answered, doesn't this v-focus get called on this element again?

Comment (leem42) [reply src/components/Eligibility.vue]: Unless I'm misunderstanding something I don't believe so, on my local server I don't see the element highlight more than once

Comment (liujoshua) [reply src/components/Eligibility.vue]: Add this inside focus function
`console.debug('focus directive: ' + el.outerHTML)`

**Review by liujoshua — APPROVED** [review]
Nice cleanup

**Pull request merged**